feature/savings/src/commonMain/kotlin/org/mifos/mobile/feature/savings/savingsAccount/SavingsAccountDetailScreen.kt

```diff
@@ -89,11 +91,12 @@ private fun SavingsAccountDetailScreen(
                 is SavingsAccountDetailUiState.Loading -> MifosProgressIndicatorOverlay()
 
                 is SavingsAccountDetailUiState.Success -> {
+                    logger.e { "SavingsAccountDetail Content: ${uiState.savingAccount.getNominalAnnualInterestRate()}" }
                     if (uiState.savingAccount.status?.submittedAndPendingApproval == true) {
                         EmptyDataView(
```

Comment:
Same here as I mentioned create another state for Empty so it should look like this:
is Error ->
is Loading ->
is Empty ->
is Success ->